src/Functions.Worker.Extensions.OpenAI/Assistants/AssistantPostInputAttribute.cs

```diff
@@ -33,4 +33,14 @@ public AssistantPostInputAttribute(string id, string UserMessage)
     /// Gets user message that user has entered for assistant to respond to.
     /// </summary>
     public string UserMessage { get; }
+
+    /// <summary>
+    /// Configuration section name for the table settings for chat storage.
+    /// </summary>
+    public string? ChatStorageConnectionSetting { get; set; }
+
+    /// <summary>
+    /// Table collection name for chat storage.
+    /// </summary>
+    public string CollectionName { get; set; } = "SampleChatState";
```

Comment:
Do we really want to use a "sample" collection name as the default? If we think we need a default here, I suggest something like "ChatState" instead of "SampleChatState".

Reply:
updated to use ChatState